xonsh/completers/vox.py

```diff
@@ -0,0 +1,32 @@
+import os
+from xonsh.__amalgam__ import builtins
+from xonsh.platform import scandir
+from xonsh.vox import Vox
+
+DEFAULT_ENV_HOME = os.path.expanduser('~/.virtualenvs')
```

Comment:
Seems to be returning `AttributeError: module 'builtins' has no attribute '__xonsh_expand_path__'`
for the tests below: https://travis-ci.org/xonsh/xonsh/jobs/142168766#L211

Any idea what's causing this?